Don't use eigen private namespaces (breaks depending on eigen version)

--- src/spatial/explog.hpp~	2018-06-01 18:16:43.000000000 +0200
+++ src/spatial/explog.hpp	2018-07-11 11:28:09.422762382 +0200
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2015-2017 CNRS
+// Copyright (c) 2015-2018 CNRS
 // Copyright (c) 2015 Wandercraft, 86 rue de Paris 91400 Orsay, France.
 //
 // This file is part of Pinocchio
@@ -38,7 +38,7 @@
   ///
   /// \return The rotational matrix associated to the integration of the angular velocity during time 1.
   ///
-  template <typename D> Eigen::Matrix<typename D::Scalar,3,3,Eigen::internal::traits<D>::Options>
+  template <typename D> Eigen::Matrix<typename D::Scalar, 3, 3>
   exp3(const Eigen::MatrixBase<D> & v)
   {
     EIGEN_STATIC_ASSERT_SAME_VECTOR_SIZE (Eigen::MatrixBase<D>,
@@ -48,7 +48,7 @@
     if (nv > 1e-14)
       return Eigen::AngleAxis<typename D::Scalar>(nv, v / nv).matrix();
     else
-      return Eigen::Matrix<typename D::Scalar,3,3, Eigen::internal::traits<D>::Options>::Identity();
+      return Eigen::Matrix<typename D::Scalar, 3, 3>::Identity();
   }
 
   /// \brief Same as \ref log3
